todoapp/app.py

```diff
@@ -49,10 +56,24 @@ def create_app():
     app.config['SECRET_KEY'] = 'SFGBSRTHRYHDSFVSGBTYHYWG'
     app.config['SESSION_TYPE'] = 'filesystem'
 
+    if app.config['LOGGLY_TOKEN'] is not None:
+        logglyUrl = f'https://logs-01.loggly.com/inputs/{app.config["LOGGLY_TOKEN"]}/tag/todo-app'
+
+        handler = HTTPSHandler(logglyUrl)
+        handler.setFormatter(Formatter("[%(asctime)s] %(levelname)s in %(module)s: %(message)s"))
+        app.logger.addHandler(handler)
+
+        handler = HTTPSHandler(logglyUrl)
+        handler.setFormatter(jsonlogger.JsonFormatter())
```

Comment:
If this was the attempt at the "structured logging" stretch goal, it's slightly missed the goal of structured logs. We want to make it easy to programmatically search the logs.

E.g. how would you find logs related to a particular item or action? Or how would you count similar actions etc
You don't want to search for logs with regex expressions to analyse the string.

The idea is that instead of just logging a string, you can log key value pairs. E.g. `app.logger.info("Item updated", item_id=item_id, user_id=user_id)` to generate easy to use json data instead.